pkg/metronome/client.go

```diff
@@ -0,0 +1,54 @@
+package metronome
+
+import (
+	"encoding/json"
+
+	"github.com/dcos/dcos-cli/pkg/httpclient"
+)
+
+
+// Constants for embed query parameters
+const embedActiveRuns = "activeRuns"
+const embedSchedules = "schedules"
+const embedHistory = "history"
+const embedHistorySummary = "historySummary"
+
+// Client is a client for Cosmos.
+type Client struct {
+	http *httpclient.Client
+}
+
+// NewClient creates a new Metronome client.
+func NewClient(baseClient *httpclient.Client) *Client {
+	return &Client{
+		http: baseClient,
+	}
+}
+
+// Jobs returns a list of all job definitions.
+func (c *Client) Jobs() ([]Job, error) {
+
+	req, err := c.http.NewRequest("GET","/v1/jobs", nil, httpclient.FailOnErrStatus(true))
+	if err != nil {
+		return nil, err
+	}
+
+	// Add embed query parameters to the request URL
+	embedForJobsHistory := []string{embedActiveRuns, embedSchedules, embedHistorySummary}
```

Comment:
Should this be configurable? Eg. accepting a struct as a parameter indicating what embed fields are needed.